internal/runtimehandlerhooks/utils.go

```diff
@@ -121,3 +124,82 @@ func UpdateIRQSmpAffinityMask(cpus, current string, set bool) (cpuMask, bannedCP
 	}
 	return maskStringWithComma, invertedMaskStringWithComma, nil
 }
+
+// cpuMaskToCPUSet parses a CPUSet received in a Mask Format, see:
+// https://man7.org/linux/man-pages/man7/cpuset.7.html#FORMATS
+func cpuMaskToCPUSet(cpuMask string) (cpuset.CPUSet, error) {
```

Comment:
Same here, isn't cpuset providing this already?

Reply:
We need the set, e.g 0-7, as a mask  "ff"